### PR TITLE
chore(curriculum): replace transcripts with videos

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [ ] I have tested these changes either locally on my machine, or GitPod.

Related to #57695

This replace the transcripts for:

- What Are Switch Statements and How Do They Differ from If/Else Chains?
- What Is the Purpose of Functions, and How Do They Work?
- What Are Arrow Functions, and How Do They Work?
- What Is Scope in Programming, and How Does Global, Local, and Block Scope Work?
- What Is the Map Method, and How Does It Work?
- How Can You Match and Replace All Occurrences in a String?

### CODE REVIEW

**Comment by Dario-DC** [comment]
I've just noticed that a video is still set to private and another has a small repetition that we might want to fix. Keeping this as a draft for now.

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md [hunk 1]
```diff
@@ -1,76 +1,14 @@
 ---
 id: 673284e7244c0c1a649121b9
 title: What Is Scope in Programming, and How Does Global, Local, and Block Scope Work?
-challengeType: 19
-# videoId: nVAaxZ34khk
+challengeType: 11
+videoId: saSaw-VOFuo
```
Comment (zairahira) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md:5]: This one is set to private.

Comment (Dario-DC) [reply curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md:5]: I know, I'm waiting for Beau to be set it unlisted

Comment (jdwilkin4) [reply curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md:5]: I am able to set it to unlisted on my end. But do we know if there is a particular reason why it is private?
Like is it not ready or was it marked private by accident? Did Beau say?

Comment (jdwilkin4) [reply curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md:5]: Just finished watching it and it seems fine. So my guess is that it was marked private by accident.
It is unlisted now

Comment (Dario-DC) [reply curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284e7244c0c1a649121b9.md:5]: I think is that it's been set private by accident

**Comment by Dario-DC** [comment]
The repetition is [here](https://www.youtube.com/watch?v=WF-fbEMJ1XM) at 0.48
I would like to have it fixed but it's not a big deal @jdwilkin4

**Comment by jdwilkin4** [comment]
Yeah, I vote we fix it and remove it from this PR from now.
Because I can see this issue reported by a camper and we will need to fix it anyway

**Comment by Dario-DC** [comment]
Beau says he's going to fix it. About the private video, he's investigating. Apparently is not set private but it is to me.

**Comment by jdwilkin4** [comment]
> About the private video, he's investigating. Apparently is not set private but it is to me.

It was originally marked draft. 
So that is why it was private to you 👍

File: curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284d5e52ef81a2169b097.md [hunk 2]
```diff
@@ -1,112 +1,14 @@
 ---
 id: 673284d5e52ef81a2169b097
 title: What Are Arrow Functions, and How Do They Work?
-challengeType: 19
-# videoId: nVAaxZ34khk
+challengeType: 11
+videoId: WF-fbEMJ1XM
```
Comment (Dario-DC) [inline curriculum/challenges/english/25-front-end-development/lecture-working-with-functions/673284d5e52ef81a2169b097.md:5]:
```suggestion
videoId: EVrZqQ7Y3So
```

**Comment by Dario-DC** [comment]
Still not able to see that video but it seems anyone else can so I guess we can include it in this PR

**Review by jdwilkin4 — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**